HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -203,10 +203,22 @@ def customiseFor2018Input(process):
     process = synchronizeHCALHLTofflineRun3on2018data(process)
 
 
+def customizeToDropObsoleteMessageLoggerOptions(process):
```

Comment:
Please add the PR number in the name or in a comment.

Reply:
There is no pull request yet which requires this change (that will only happen after all these changes go through), or do you mean you want the pull request number for this pull request?

Reply:
Usually it is this PRs number, to remind why this customisation was introduced when I remove it later after a new release has been parsed into ConfDB.